Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.cpp

```diff
@@ -147,6 +147,15 @@ void ABenchmarkGymGameModeBase::BeginPlay()
 	{
 		OutputPlayerDensity();
 	}
+
+	if (bLongFormScenario)
+	{
+		// Extend timers to handle longer expected deployment lifetime (required for current long form disco performance test)
+		RequiredPlayerCheckTimer.SetTimer(17 * 60);
+		DeploymentValidTimer.SetTimer(38 * 60);
+		UNFRConstants* NFRConstants = const_cast<UNFRConstants*>(UNFRConstants::Get(GetWorld()));
```

Comment:
Feel as if there could be room for a mini rework of how UNFRConstants work but... maybe in future.

Reply:
Definitely agree

Reply:
+1